### ISSUE TITLE
Provide changelogs

### ISSUE THREAD

**Issue by Etim-Orb** [foobnix/LibreraReader#783]
Make the full list of changes available either in the GitHub Releases section or update them on the website which is not up to date.

**Comment by GroundJet** [comment]
Please note that we understand writing code can be difficult as it is and writing changelogs and documentation on top of that can be extra work but it would be greatly appreciated.
Also certain commits can be cryptic like say https://github.com/foobnix/LibreraReader/commit/f6444a9c0265c3a8c14cceddc29c95e02e0b5334 where the only commit message is 'hope to fix error', a more descriptive commit message could help us volunteer to write changelogs
Thank you.

**Comment by nach0** [comment]
Yes please. On https://librera.mobi/what-is-new/ we can see changes for some versions, but not for others. On the GitHub releases there is also no information for some versions (others have some text). On F-Droid we don't get all versions, and frequently I get an update without any changelog information.
